package.json

```diff
@@ -23,6 +23,7 @@
     "npm": "6"
   },
   "homepage": "https://github.com/bigcommerce/checkout-sdk-js",
+  "prettier": "@bigcommerce/eslint-config/prettier",
```

Comment:
Is this line working? I tried it in `checkout-js`, it was like it did not agree with `"@bigcommerce/eslint-config": "^1.0.1"`.

Reply:
Good catch, could be left over from 2.6.1 testing

Reply:
yep just leftover from 2.6.1